.appveyor.yml

```diff
@@ -0,0 +1,59 @@
+version: "{build}"
+clone_folder: "C:\\WORK"
+
+environment:
+  global:
+    CABAL: "cabal --store-dir=C:\\SR --http-transport=plain-http"
+    CABAL_VERSION: "2.4.1.0"
+    HC: "ghc"
+    TEST: "--enable-tests"
+    BENCH: "--enable-benchmarks"
+  matrix:
+    - GHC_VERSION: "8.6.4"
+    - GHC_VERSION: "8.6.3"
+    - GHC_VERSION: "8.6.2"
+    - GHC_VERSION: "8.6.1"
+    - GHC_VERSION: "8.4.4"
+    - GHC_VERSION: "8.4.3"
+    - GHC_VERSION: "8.4.2"
+    - GHC_VERSION: "8.4.1"
+    - GHC_VERSION: "8.2.2"
+    - GHC_VERSION: "8.0.2"
+    - GHC_VERSION: "8.0.1"
+    - GHC_VERSION: "7.10.3"
+    - GHC_VERSION: "7.10.2"
+
+cache:
+ - "C:\\SR"
+
+install:
+ - "choco install -y ghc --version %GHC_VERSION%"
+ - "choco install -y cabal --version %CABAL_VERSION%"
+ - "refreshenv"
+ - "set PATH=C:\\msys64\\mingw64\\bin;C:\\msys64\\usr\\bin;%PATH%"
```

Comment:
Not sure if we need to set those mingw64 env variables since GHC on Windows uses its own mingw64 binaries.

Reply:
I'll try remove them

Reply:
Maybe it was so we can use posix shell commands in the script?

Reply:
Maybe yes

Reply:
I've removed it, let's see what happens